build.gradle

```diff
@@ -22,21 +42,50 @@ plugins {
 }
 
 
+apply from: 'gradle/formatting.gradle'
+apply plugin: 'maven-publish'
+apply plugin: 'opensearch.opensearchplugin'
+apply plugin: 'opensearch.testclusters'
+
+opensearchplugin {
```

Comment:
seems that replacing the application gradle plugin with the opensearch gradle plugin breaks `./gradlew run` and does not execute the configured main class of the opensearch gradle plugin.

CC : @owaiskazi19

Reply:
@ryanbogan can verify the above? We had `application` plugin before to run the SDK with defined `MainClassName`.

Reply:
we cannot have both plugins as `application` and `opensearchplugin` both define a `run` task

Reply:
@joshpalis I didn't say we can have both :slightly_smiling_face:. Since I added `application` plugin I'm aware of that. That comment was just to explain the usage of `application`.

Reply:
Yeah we ran into this issue yesterday - I'm not sure why it's not seeing the class name to run the extension.

Reply:
I tried a couple of approaches which didn't help either:
1. Created a main file under `org.opensearch.sdk` instead of using HelloWorldExtension.
2. Included/excluded few dependencies which are added in this PR.
3. Tried debugger for `main` class. `opensearchplugin` doesn't invoke the process at all.

@dbwiddis @saratvemulapalli any idea here?